homework/final/Quicksort/MultithreadedQuicksort/ThreadedQuicksort/Program.cs

```diff
@@ -0,0 +1,41 @@
+﻿// See https://aka.ms/new-console-template for more information
+
+using System.Diagnostics;
+using System.Runtime.InteropServices.JavaScript;
+using MultiThreadedQuicksort.ThreadedQuicksort;
+
+const int min = 0;
+const int max = 1000;
+const int size = 10240;
+const int singleThread = 1;
+
+var multiThreadedQuicksort = new ThreadedQuicksort<int>();
+var singleThreadedQuicksort = new ThreadedQuicksort<int>(singleThread);
+
+var randNum = new Random();
+var arrayForMultiThreaded = Enumerable
+    .Repeat(0, size)
+    .Select(i => randNum.Next(min, max))
+    .ToArray();
+var arrayForSingleThreaded = new int[size];
+arrayForMultiThreaded.CopyTo(arrayForSingleThreaded, 0);
+
+var multiThreadedWatch = new Stopwatch();
+multiThreadedWatch.Start();
+var multiThreaded = multiThreadedQuicksort.Quicksort(arrayForMultiThreaded);
+await multiThreaded.ConfigureAwait(false);
+
```

Comment:
```suggestion
multiThreadedWatch.Stop();

```
Лооооол, таймер после замера надо остановить же.